src/ontology/uberon-edit.obo

```diff
@@ -224493,6 +224493,25 @@ relationship: dc-contributor http://orcid.org/0000-0001-7258-9596
 relationship: has_part UBERON:0001384 ! primary motor cortex
 relationship: has_part UBERON:0016634 ! premotor cortex
 
+[Term]
+id: UBERON:8440078
+name: teat
+def: "The projection from the mammary glands of mammals from which milk flows or is ejected for the purpose of feeding young. As compared to a nipple, where the individual lactiferous ducts open directly to the outside world, a teat has lactiferous ducts that converge on a large lactiferous sac at the base of the teat and there is a single, large duct that runs through the teat with an outlet at the end of the teat." [https://orcid.org/0000-0002-1572-1316, WikipediaVersioned:Teat&oldid=1107850923]
+comment: While many lay sources treat nipple and teat as synonyms, mammalogists do not. {xref="https://orcid.org/0000-0002-1572-1316"}
+is_a: UBERON:0000475 ! organism subdivision
+relationship: dc-contributor http://orcid.org/0000-0001-7258-9596
+property_value: editor_note "Taxon restriction should be added on this term when possible." xsd:string
+property_value: seeAlso https://github.com/obophenotype/uberon/issues/2353 xsd:string
+
+[Term]
+id: UBERON:8440079
+name: teat skin
+def: "The skin that is part of the teat." [https://orcid.org/0000-0001-7258-9596]
+intersection_of: UBERON:0001003 ! skin epidermis
```

Comment:
general red flag here: label, text def, and logical def should be inconcordance

*in general* in uberon, skin = epidermis + dermis, so we shouldn't conflate the two - even if there are some zones where dermis is lacking.

Also I don't think we can address this issue coherently until we resolve #1873. If teat/nipple is a projection of the skin, as proposed there, then teat skin = teat!

We really need clear boundaries defined for all terms